README.md

```diff
@@ -1 +1,77 @@
-# hello-microverse
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# hello-microverse
+
+> This is a site showing hello microverse text as a header.
+
+![screenshot](./images/pic.png)
```

Comment:
This image is not working, you should remove it or fix it as you want.

- [ ] Remove or Fix the image in your README file.